source/contribute.rst

```diff
@@ -166,7 +166,36 @@ Now do some work, make some changes then commit them:
 .. code-block:: console
 
     $ git status
-    $ git commit <filespec>
+    $ git add -p
+    $ git commit -v
+
+Commit messages should look like emails, meaning they should have a
+subject, but also a body. The subject should be on the first line, and
+not exceed 50 chars. It should tell us what you did, and every change in
```

Comment:
I have a question: why is this comma is correct?

Reply:
It is a pause - if I read this without pauses, it is very awkward to listen to.

I don't know the grammatical rule behind it, sorry :-(